CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Hasura GraphQL Engine Changelog
 
-## Next release
+## `v1.3.0-beta.4`
```

Comment:
Could you also add a new section for `Next Release` :) ?

Something like this:

```
## Next release

### Bug fixes and improvements

(Add entries here in the order of: server, console, cli, docs, others)
```